app/services/monitor_index_jobs.rb

```diff
@@ -0,0 +1,65 @@
+# Monitor the Done jobs from the SQS queue
+class MonitorIndexJobs
+  def initialize
+    @todo_jobs_queue     = TodoJobsQueue.new
+    @done_jobs_queue     = DoneJobsQueue.new
+    @dead_queue          = DeadLetterJobsQueue.new
+    @processed_from_done = 0
+    @desired_capacity_reset_by = 0
+    @worker_asg          = OpenStax::Aws::AutoScalingGroup.new(
+      name: Rails.application.secrets[:search_worker_asg_name],
+      region: ENV.fetch('REGION') )
+  end
+
+  def call
+    process_done_jobs
+    notify_sentry_for_dead_letter_queue
+    reset_desired_capacity_if_needed
+
+    stats
+  end
+
+  private
+
+  def notify_sentry_for_dead_letter_queue
+    loop do
+      dead_job = @dead_queue.read
+      break if dead_job.nil?
+
+      Raven.capture_message("Job Found in Dead Letter Queue", :extra => dead_job.inspect)
```

Comment:
Should probably also have a little log message that we are sending to sentry (unless sentry does that for us) to be consistent with the other private method logging.  Don't need/want to log the `.inspect` content, just say that we are sending something to sentry, maybe include the index name or something identifiable.